libopenage/renderer/window.h

```diff
@@ -35,6 +44,8 @@ struct window_settings {
 	bool vsync = true;
 	// If true, enable debug logging for the selected backend.
 	bool debug = false;
+	// Display mode for the window.
+	window_mode mode = window_mode::FULLSCREEN;
```

Comment:
```suggestion
	window_mode mode = window_mode::WINDOWED;
```

The default should be windowed as this has been the previous default.